Assets/Scripts/BirdScript.cs

```diff
@@ -0,0 +1,293 @@
+    // Start is called before the first frame update
+
+
+    void Awake()
+    {
+        scriptLoaded = SceneManager.GetActiveScene().name;
+        setupGibberish();
+        WorkPanels.gameObject.SetActive(false);
+    }
+
+    // Update is called once per frame
+    void FixedUpdate()
+    {
+
+        switch (scriptLoaded)
+        {
+            case "The Facility":
+                FacilityUpdate();
+                break;
+            case "Pond":
+                PondUpdate();
+                break;
+            case "Office":
+                OfficeUpdate();
+                break;
+            case "Lighting":
+                LightingUpdate();
+                break;
+            case "FirstEncounter":
+                FirstEncounter();
+                break;
+            case "Mumble":
+                JibberishMumbule();
+                break;
+
+
+
+
+        }
+
+        //Debug.Log(Time.fixedTime);
+        
+    }
+
+    void FacilityUpdate()
+    {
+
+        if (Time.fixedTime == 1)
+        {
+            WorkPanels.gameObject.SetActive(true);
+            SaidToPlayer.Invoke("That looks okay.");
+        }
+            
+
+        if (Time.fixedTime == 3)
+            SaidToPlayer.Invoke("At least it doesn't look that bad.");
+
+        if (Time.fixedTime == 6.5)
+            SaidToPlayer.Invoke("I think.");
+
+        if (Time.fixedTime == 10)
+            SaidToPlayer.Invoke("And they say it was stupid complicated.");
+
+        if (Time.fixedTime == 20)
+            SaidToPlayer.Invoke("That should be just about all done.");
+
+        if (Time.fixedTime == 22)
+            SaidToPlayer.Invoke("Crap, forgot about the droids.");
+
+    }
+
+    void PondUpdate()
+    {
+        if (Time.fixedTime == 1)
+            SaidToPlayer.Invoke("Hi again.");
+
+        if (Time.fixedTime == 3)
+            SaidToPlayer.Invoke("Welcome to the pond.");
+
+        if (Time.fixedTime == 6.5)
+            SaidToPlayer.Invoke("This is where I come to think.");
+
+        if (Time.fixedTime == 10)
+            SaidToPlayer.Invoke("I always find myself a bit more realaxed in a natural setting.");
+
+        if (Time.fixedTime == 14)
+            SaidToPlayer.Invoke("Even if it is simulated.");
+
+        if (Time.fixedTime == 16)
+            SaidToPlayer.Invoke("Like listening to rainstorms while you sleep.");
+
+        if (Time.fixedTime == 20)
+            SaidToPlayer.Invoke("Except I hate rain.");
+
+        if (Time.fixedTime == 22)
+            SaidToPlayer.Invoke("Because of the whole get wet deal.");
+
+        if (Time.fixedTime == 25)
+        {
+            WorkPanels.gameObject.SetActive(true);
+            SaidToPlayer.Invoke("Look around for a bit, I have some backstage work to get done here.");
+        }
+            
+
+        if (Time.fixedTime == 26)
+        {
+            eventStartTime = Time.fixedTime;
+            scriptLoaded = "Mumble";
+        }
+            
+    }
+
+    void OfficeUpdate()
+    {
+
```

Comment:
Remove unused functions.

Reply:
this is intended as i haven't written the script for office yet